_targets.R

```diff
@@ -26,17 +42,13 @@ p2_targets_list <- list(
   tar_target(
     site_data_annotated,
     annotate_data(site_data_clean, site_filename = site_info_csv)
-  ),
-  tar_target(
-    site_data_styled,
-    style_data(site_data_annotated)
```

Comment:
This isn't explicitly stated in the instructions, so I think what you have here is great - it does combine some of the individual steps into one. However, I think we can take this one step further and have our `process_data()` function absorb what is now in `annotate_data` and here's why: every individual target we define requires a certain amount of overhead (usually resulting in slower builds) as the pipeline checks whether a target is outdated or not when you go to build. The more targets we have, the slower the build will be. So, we need to be very strategic about right-sizing the targets we add. Right-sizing them is challenging and a skill that we all develop the more we build these. Here are some questions I think about before deciding whether I make it one or multiple targets:

1. **Necessity of each individual target.** Do I need to use both targets in downstream steps (keep them separate) OR is one just an intermediate stage that will not be used in that exact form later (combine them into one)?
2. **Processing time and potential for unnecessary, lengthy rebuilds.** Are either of the steps in question lengthy to run? If so, are there potential parameterizations in the second, where changes to the inputs aren't impacting the first step (and that step takes a long time to run), but would require the second to rebuild? If so, keep them separate because it will save you build time as you iterate on those parameterizations.